<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningModeA</class>
 <widget class="QDialog" name="LearningModeA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Learning mode A</string>
   </property>
  </widget>
  <widget class="QLabel" name="listName">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageAName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>language1</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageAWord">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>wordLanguage1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="revealButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Reveal</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageBWord">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>wordLanguage2</string>
   </property>
  </widget>
  <widget class="QLabel" name="languageBName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>language2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousWordButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Previous word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextWordButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Next word</string>
   </property>
  </widget>
  <widget class="QLabel" name="commentText">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>360</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>comment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
